digitalpy/core/zmanager/impl/default_routing_worker.py

```diff
@@ -192,24 +196,32 @@ def process_next_request(self, controllerClass, response):
         # nextRequest.set_response_format(request.get_response_format())
         self.action_mapper.process_action(nextRequest, response)
 
-    def receive_request(self):
-        message = self.sock.recv_multipart()
-        topic = message[0]
+    def receive_request(self) -> Tuple[str, Request]:
+        """Receive and process a request from the ZMQ socket.
 
-        print("received topic %s" % topic)
-
-        topic_section = topic.decode("utf-8").split("/")
-
-        request_id = topic_section[7]
-
-        response_topic = f"/routing/response/{topic_section[3]}/{topic_section[4]}/{topic_section[5]}/{topic_section[6]}/{request_id}"
-
-        request = ObjectFactory.get_new_instance("request")
-        request.values = message[1]
-        request.set_sender(topic_section[3])
-        request.set_context(topic_section[4])
-        request.set_action(topic_section[5])
-        request.set_format(topic_section[6])
-
-        self.formatter.deserialize(request)
-        return topic_section,response_topic,request
+        Returns:
+            A tuple containing the topic sections as a list, the response topic as a string, and the request object.
+        """
+        try:
+            # Receive message from client
+            message = self.sock.recv_multipart()[0]            
+            sender, context, action, format, protocol, values = message.split(b",", 5)
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WPS236 Found too many variables used to unpack a tuple: 6 > 4